MAUI/Maps/tile-layer.md

```diff
@@ -0,0 +1,216 @@
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![TomTom maps default view](images/tile-layer/tom_tom_default.png)
+
+## Changing the center latitude and longitude
+
+You can set the center position by setting the `MapTileLayer.Center` property. It represents the center position of the map layer.
+
+Based on the size of the [`SfMaps`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.html) control, `Center` and `ZoomLevel` number of initial tiles needed in the view port alone will be rendered. Refer this section for enabling `zooming and panning`.
+
+Defaults to `MapLatLng(0.0, 0.0)`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate="https://tile.openstreetmap.org/{z}/{x}/{y}.png">
+            <map:MapTileLayer.Center>
+                <map:MapLatLng Latitude="27.175014"
+                               Longitude="78.042152">
+                </map:MapLatLng>
+            </map:MapTileLayer.Center>
+        </map:MapTileLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png";
+tileLayer.Center = new MapLatLng(27.175014, 78.042152);
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![OSM map center](images/tile-layer/osm_center.png)
+
+## Cache a tile images in application memory
+
+The `CanCacheTiles` property used to decide whether the tile images should be cached in application memory or not. The default value of the `CanCacheTiles` is `false`.
+
+While enabling the `CanCacheTiles` as `true` then need to set the tile server name to maintain the folder to store cache tiles in `MapTileLayer.UrlTemplate` property. The default tile server name to store tile cache is OSM. If you want to change server name you can use the below URL format.
+
+https://example_provider/{z}/{x}/{y}.png?name=serverName
+
+Here, you can replace the serverName as per your wish.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate = "http://api.tomtom.com/map/1/tile/basic/main/{z}/{x}/{y}.png?key=subscription_key?name=tomtom"
+                          CanCacheTiles= "True">
+        </map:MapTileLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "http://api.tomtom.com/map/1/tile/basic/main/{z}/{x}/{y}.png?key=subscription_key?name=tomtom";
+tileLayer.CanCacheTiles = true;
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Clear a cached tile images from application memory
+
+The `DeleteTilesFromCache` method used to clear the cached tile images from  application cache memory.
```

Comment:
The `DeleteTilesFromCache` method is used to clear the cached tile images from the application cache memory.

Reply:
Modified the content